ui/app/AppLayouts/Communities/panels/MembersTabPanel.qml

```diff
@@ -21,6 +21,10 @@ Item {
     property string placeholderText
     property var model
     property var rootStore
+    property int memberRole: Constants.memberRole.none
+
+    property bool isOwner: memberRole === Constants.memberRole.owner
+    property bool isTokenMaster: memberRole === Constants.memberRole.tokenMaster
```

Comment:
`readonly` those 2

Reply:
Also could be declared inside `memberItem` instead, for the clarity of it being private.

Reply:
Since `memberItem` is a delegate, I think it's better to keep it here so that those properties are not initiated for each member. I'll put the `readonly` though.